great_expectations/data_context/types/base.py

```diff
@@ -1513,10 +1513,10 @@ class Meta:
         allow_none=True,
     )
     # Next fields are used by configurators
```

Comment:
Same thing here -- please add the number of fields used for whichever purpose (for an easier code readability).